### ISSUE TITLE
Would it be viable to implement an option to have dos style music? Apologies if the ability to do this is already there or if this is inappropriate. 

### ISSUE THREAD

**Issue by goat-soup-eater** [1oom-fork/1oom#6]

**Comment by oblivioncth** [comment]
You need a copy of the game that was built with Fluidsynth support and then you provide a "sound font" as a config option or command line argument

See: https://www.reddit.com/r/masteroforion/comments/zock5e/how_to_buildrun_1oom_with_soundfont_fluidsynth/

I prefer to use the following sound font which mimics a Soundblaster 16. This may be the "dos style" music you're referring to. I think the music produced by this source port by default is based on the Roland MT-32.

**Comment by 1oom-fork** [comment]
> You need a copy of the game that was built with Fluidsynth support and then you provide a "sound font" as a config option or command line argument
> 
> See: https://www.reddit.com/r/masteroforion/comments/zock5e/how_to_buildrun_1oom_with_soundfont_fluidsynth/
> 
> I prefer to use the following sound font which mimics a Soundblaster 16. This may be the "dos style" music you're referring to. I think the music produced by this source port by default is based on the Roland MT-32.

If you want to participate in the development of the project, please make suggestions about existing documentation and do not post links to broken manuals. Thank you.

**Comment by oblivioncth** [comment]
> If you want to participate in the development of the project, please make suggestions about existing documentation and do not post links to broken manuals. Thank you.
There is nothing broken about the guide, other than a small typo in the invocation of `make` that has been corrected. This repository's complication instructions are written with the assumption that reader has at least minor previous experience with compiling software. There's nothing wrong with this itself, however, it does not help people such as the author (I'm assuming here) who originally were just grabbing a pre-built release (i.e. not compiling themselves) and simply want fluidsynth support.

Additionally, the existing compiling instructions do not result in a build with functional fluidsynth for me, though I don't have time at this moment to double check if I made a mistake or delve into what's missing and will get around to that when I can. There are also minor possible points of confusion, such as the fact that the instructions specify to check the libfluidsynth x86 release for which DLLs are required, but does not specify where to copy them from and that referenced release package includes some DLLs that are not available within MINGW32 (at least not without potentially installing more packages).

I already offered to make the Windows compiling instructions more comprehensive but you rejected those changes.

In this situation, I don't think linking to guide that covers all steps from start to finish and works is at all inappropriate. Perhaps it's a _hair_ out of place given this is an issue tracker and would ultimately fit better in a Discussion (along with the original question), but that feature is not enabled on this repository.

**Comment by 1oom-fork** [comment]
TLDR
I still hope you can follow some rules and make suggestions instead of wasting your time
Thank you

**Comment by 1oom-fork** [comment]
The discussions tab is open.
Thanks for the feedback

**Comment by 1oom-fork** [comment]
Posted v1.7.1 release with FluidSynth support for those who don't like v1.7.6a

**Comment by 1oom-fork** [comment]
I just unzip everything into the 1oom folder.

**Comment by 1oom-fork** [comment]
hw.sdlmixersf = "./YOUR_SOUND_FONT.sf2"
This is not the correct syntax for Windows.
Replace each '/' with '\\'

**Comment by goat-soup-eater** [comment]
[1oom_log.txt](https://github.com/1oom-fork/1oom/files/11274028/1oom_log.txt)

I have the DLLs extracted and the log thing corrected. I'm swapping between two different soundfonts but there doesn't seem to be a difference, so I'm assuming it's not working.

**Comment by 1oom-fork** [comment]
SDLA: no soundfonts
Fix your paths as stated in my previous post

**Comment by goat-soup-eater** [comment]
[1oom_config_game_classic_sdl2.txt](https://github.com/1oom-fork/1oom/files/11274084/1oom_config_game_classic_sdl2.txt)

**Comment by oblivioncth** [comment]
Windows supports the forward slash as a path separator in many contexts and it's what I use in my working config file with a 1.7.1 build.

**Comment by 1oom-fork** [comment]
Added warning about invalid sound font path:
https://github.com/1oom-fork/1oom/commit/57c468349af146bcbe5636b04cb406a1fab2cd93

**Comment by 1oom-fork** [comment]
You can try: 'cmd.exe' -> 'perfmon /res' ->CPU -> Associated Modules (and check 1oom.exe box, so you will see in the list if fluidsynth is connected)
Added warning about invalid sound font path:
https://github.com/1oom-fork/1oom/commit/57c468349af146bcbe5636b04cb406a1fab2cd93
Added v1.7.6b release.

**Comment by oblivioncth** [comment]
> To oblivioncth: I hope we've fixed the path format confusion

I'm uncertain of exactly what the confusion was about as I was just commenting that in this case `/` and `\\` both work fine. I use `hw.sdlmixersf = "./SB16.sf2"` in my primary Windows copy.

Combining the [1oom-fork-v1.7.6b-SDL2-fluidsynth.zip](https://github.com/1oom-fork/1oom/releases/download/1oom-fork-v1.7.6/1oom-fork-v1.7.6b-SDL2-fluidsynth.zip) release along with the latest 32-bit fluidsynth binaries as noted above does work correctly on my system. Might have just missed a DLL when trying to compile it as I hastily copied them one-by-one from the MSYS package directory instead of using the ones from the Fluidsynth release.

**Comment by 1oom-fork** [comment]
About confusion:
In fact, even a pure "path" without "./" works for me.
There is no warning so you don't know what's wrong.

**Comment by oblivioncth** [comment]
> About confusion: In fact, even a pure "path" without "./" works for me. There is no warning so you don't know what's wrong.

Makes sense since Windows regards that as relative. I was curious if perhaps the Unix library wrappers of MinGW might have changed that, but clearly this isn't the case.

The addition of the path certainly makes troubleshooting easier.
